R/reactive_values.R

```diff
@@ -8,31 +8,35 @@ setup_reactive_values_obj <- function(input) {
         stringi::stri_extract_first(str = infile$name, regex = ".*")
     }
 
-    load_data <- function(in_file, two_datasets=NULL) {
+    load_data <- function(in_file, two_datasets=NULL, is_design_file = FALSE) {
         infile <- in_file
         if (is.null(infile) || (is.logical(two_datasets) && two_datasets==FALSE)) {
             return(NULL)
         }
         raw_df <- read_tsv(infile$datapath, col_types = cols())
 
-        original_colnames <- colnames(raw_df)
-        corrected_colnames <- make.names(colnames(raw_df))
 
-        colnames(raw_df) <- make.names(colnames(raw_df))
+        if (is_design_file) {
+            raw_df <- raw_df %>%
+              mutate_all(make.names) # Run make.names on all elements of the design matrix
```

Comment:
Just on the wording here "all elements" -> "all columns" ? Or "variable" which seems to be the syntax dplyr use

Reply:
Good point, changed it to variables!